docs/zh/quick-start.md

```diff
@@ -48,18 +45,17 @@ apisix-java-plugin-runner 与 APISIX 用 `Unix Domain Socket` 进行进程间通
 构建完成，你会在 `dist` 目录看见构建产物
 
 ```
-apache-apisix-java-plugin-runner-0.1.0-bin.tar.gz
+apache-apisix-java-plugin-runner-${your_plugin_version}-bin.tar.gz
 ```
 
 在`dist`目录添加`Dockerfile`文件
 
 ```dockerfile
-FROM apache/apisix:${version}-alpine
-
-RUN sed -i "s/dl-cdn.alpinelinux.org/mirrors.aliyun.com/g" /etc/apk/repositories && apk add --no-cache openjdk8-jre
+FROM apache/apisix:${version}-debian
 
-ADD apache-apisix-java-plugin-runner-0.1.0-bin.tar.gz /usr/local/
+RUN apt -y install openjdk-11-jdk
```

Comment:
could we use the base image with jdk 11?

Reply:
I use the image of  apisix , and jdk should in the same image. If i use the base image with jdk , I have to install apisix .